clusterloader2/pkg/prometheus/manifests/exporters/metrics-server/resourceConsumer.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
```

Comment:
As it might be up to 80 replicas, can we deploy it only when `--prometheus-scrape-metrics-server` is set to true?

Reply:
oh nvm, I forgot it's already in directory that is deployed conditionally.

Reply:
yeah, only deploy it only when` --prometheus-scrape-metrics-server` is set to true